src/components/SearchBar/SearchBar.js

```diff
@@ -128,18 +128,18 @@ const SearchBar = ({
 }
 
 /**
- * Function to render the SearchBarPopupItem component.
+ * Component that renders the SearchBarPopupItem component, which displays the searchbar items within a popup.
  *
- * @function
+ * @component
  *
- * @param {int} index - Parameter used as the index key for the SearchBarPopupItem.
- * @param {string} value - Parameter that is used as an value for a query that is used within a link in the rendered component
- * @param {string} filterQuery - Parameter that is used to filter the query based on its value.
- * @param {boolean} filter - If this parameter is true, certain elements will be shown.
- * @param {int} arrayLength - Parameter that has the length of the array as value.
- * @param {int} dataIndex - Parameter used as the data-index value of an link tag.
- * @param {boolean} setSearchBarPopupOpen - Parameter that if set to true the SearchBarPopup component will be shown.
- * @param {object} item - Parameter that its value of is used for an element.
+ * @param {int} index - Parameter that contains the index value.
+ * @param {string} value - Parameter that contains a value used in a query.
+ * @param {string} filterQuery - Parameter that contains the filter that is used within the query.
+ * @param {boolean} filter - Parameter containing a boolean value to display the name or value of the search results.
+ * @param {int} arrayLength - Parameter that contains the array length.
+ * @param {int} dataIndex - Parameter that contains the data index value.
+ * @param {function} setSearchBarPopupOpen - Function that is set in the parent state.
```

Comment:
The description suggests that the function is set in the parent state, but what does that really mean? A better description would be `Function to set the searchBarPopupOpen state`